include/swift/AST/DiagnosticsSema.def

```diff
@@ -825,6 +825,11 @@ ERROR(argument_out_of_order_unnamed_unnamed,none,
       "unnamed argument #%0 must precede unnamed argument #%1",
       (unsigned, unsigned))
 
+ERROR(member_shadows_global_function,none,
+      "type member %0 shadows %1 %0 in %2 '%3'"
```

Comment:
Additionally, the full name of the member may be different than the full name of the thing being shadowed, so reusing `%0` is incorrect.